packages/patternfly-4/react-integration/demo-app-ts/src/Demos.ts

```diff
@@ -86,10 +116,50 @@ export const Demos: DemoInterface[] = [
     componentType: Examples.DataListDemo
   },
   {
-    id: 'context-selector-demo', 
-    name: 'ContextSelector Demo', 
+    id: 'context-selector-demo',
+    name: 'ContextSelector Demo',
     componentType: Examples.ContextSelectorDemo
   },
+  {
+    id: 'donut-chart-bottom-legend-demo',
+    name: 'Donut Chart with Bottom Aligned Legend Demo',
+    componentType: Examples.DonutBottomAlignedLegendDemo
+  },
+  {
+    id: 'donut-chart-small-bottom-legend-demo',
+    name: 'Donut Chart Small with Bottom Aligned Legend Demo',
+    componentType: Examples.DonutSmallBottomAlignedLegend
+  },
+  {
+    id: 'donut-chart-small-demo',
+    name: 'Donut Chart Small Demo',
+    componentType: Examples.DonutSmallDemo
+  },
+  {
+    id: 'donut-chart-small-right-legend-demo',
+    name: 'Donut Chart Small with Right Aligned Legend Demo',
+    componentType: Examples.DonutSmallRightAlignedLegendDemo
+  },
+  {
+    id: 'donut-chart-right-legend-color-demo',
+    name: 'Donut Chart Multi-colored with Right Aligned Legend Demo',
+    componentType: Examples.DonutRightAlignedLegendColorDemo
+  },
+  {
+    id: 'donut-chart-right-legend-demo',
+    name: 'Donut Chart with Right Aligned Legend Demo',
+    componentType: Examples.DonutRightAlignedLegendDemo
+  },
+  {
+    id: 'donut-utilization-simple-bottom-demo',
+    name: 'Donut Utilization Simple Chart, Bottom-aligned Legend Demo',
+    componentType: Examples.DonutUtilizationSimpleBottomDemo
```

Comment:
This example looks really odd with a gray background.

<img width="929" alt="Screen Shot 2019-06-26 at 10 44 03 AM" src="https://user-images.githubusercontent.com/17481322/60189794-84ae9780-97ff-11e9-9128-90c69343bf29.png">

Reply:
Fixed.